### PR TITLE
Newsletter-264: translate into Chinese

### PR DESCRIPTION
**Pull request by hulatown**

### CODE REVIEW

File: _posts/zh/newsletters/2023-08-16-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+- [LDK #2308][] 允许支付发送者在自己的支付中包含定制化的 标签-长度-数值（TLV）记录，接收者可以使用 LDK 或者兼容的实现从支付中抽取出来。这可以让使用支付发送定制化数据和元数据变得更容易。{% assign timestamp="55:15" %}
+
+{% include references.md %}
+{% include linkers/issues.md v=2 issues="27213,28008,2308" %}
+[todd expire]: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2023-August/021849.html
+[gould spj]: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2023-August/021868.html
+[spj bip]: https://github.com/bitcoin/bips/pull/1483
+[gibson spj]: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2023-August/021872.html
+[gould spj2]: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2023-August/021880.html
+[news236 spj]: /en/newsletters/2023/02/01/#serverless-payjoin-proposal
```
Comment (murchandamus) [inline _posts/zh/newsletters/2023-08-16-newsletter.md:66]:
```suggestion
[news236 spj]: /zh/newsletters/2023/02/01/#serverless-payjoin-proposal-payjoin
```

**Review by murchandamus — COMMENTED** [review]
Hey, looks mostly good. I just noticed that there was one link to an English newsletter where there was already a Chinese translation.

**Comment by hulatown** [comment]
> Hey, looks mostly good. I just noticed that there was one link to an English newsletter where there was already a Chinese translation.

Oh I missed this link. Thanks Murch!

**Review by bitschmidty — APPROVED** [review]
ACK, thank you @hulatown !

**Pull request merged**